.env.sample

```diff
@@ -1,8 +1,9 @@
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=
-POSTGRES_HOST=127.0.0.1
 POSTGRES_DB=findadoc
 SHADOW_DB=shadow
+POSTGRES_HOST=findadoc-server-db-1
```

Comment:
🤔 this is not the right path

Reply:
@ann-kilzer nope, from inside thé containers it would be `db` or `database`

Reply:
Here I am configuring it for outside the containers. I override the value inside the docker compose file

Reply:
E.g. I need this value to be the outside-the-container path